src/pages/docs/tutorials/tutorial-kitten-browser/lists/index.md

```diff
@@ -141,21 +141,25 @@ const {a: myA, b} = obj; // == const myA = obj.a, b = obj.b;
 ```
 As you saw above, you can also destructure an object received as an argument to a function:
 ```js
-url: ({index, size}) => {	// == url: (props) => { const index = props.index, size = props.size;
+url: ({index, size}) => {
+	// == const {index, size} = props;
+	// == const index = props.index, size = props.size;
```

Comment:
Let's revert this change. The comment could mislead readers if the comment is not on the same lime as destructuring.

Reply:
I accepted your suggestion.